sdk/face/ai-vision-face-rest/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+This is the first preview of the Azure Face Service client library `@azure-rest/ai-vision-face` that follows the [TypeScript Azure SDK Design Guidelines](https://azure.github.io/azure-sdk/typescript_introduction.html).
+This library replaces the package [@azure/cognitiveservices-face](https://www.npmjs.com/package/@azure/cognitiveservices-face).
+
+This package's [documentation](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/face/ai-vision-face-rest/README.md) and [samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/face/ai-vision-face-rest/samples) demonstrate the new API.
+
+### Features Added
```

Comment:
Could you highlight these `Features Added ` and `Breaking` is compared with which SDK? Considering this is the first time to release our rest client.

Reply:
The 2 sections are comparing with the [old SDK](https://www.npmjs.com/package/@azure/cognitiveservices-face). I will make the descriptions more detailed.